framework/util/page_guard_manager.h

```diff
@@ -181,6 +181,8 @@ class PageGuardManager
   private:
     size_t GetSystemPageSize() const;
     size_t GetSystemPagePotShift() const;
+    size_t InitSystemExceptionHandler() const;
```

Comment:
No definition of  InitSystemExceptionHandler()?

Reply:
Must have made that prototype and then not used it, instead making InitializeSystemExceptionContext. I'll delete.